components/camel-datasonnet/src/test/resources/simpleMapping_result.json

```diff
@@ -0,0 +1 @@
+{"count":1,"isActive":true,"testVar":"HelloWorld","uid":123,"uname":"JavaDuke","fullName": "DataSonnet"}
```

Comment:
Add ASF license please

Reply:
Oh is there a way to add license to JSON files?

Reply:
Not sure if it is feasible, probably not :-)